src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -238,7 +238,7 @@ class OPENVINO_GENAI_EXPORTS GenerationConfig {
 
     /// @brief checks that are no conflicting parameters, e.g. do_sample=true and num_beams > 1.
     /// @throws Exception if config is invalid.
-    void validate() const;
+    void validate(size_t prompt_length = 0) const;
```

Comment:
This is part of public API. Can you change src/cpp/src/continuous_batching/pipeline_impl.cpp instead to additionally assert prompt_len